packages/documentation/src/sidebar.js

```diff
@@ -267,6 +267,10 @@ module.exports = {
               name: 'Feature flags',
               href: '/platform/tools/feature-flags',
             },
+            {
+              name: 'Downtime notifications',
+              href: '/platform/tools/downtime-notifications',
+            },
```

Comment:
This made sense to me here, but LMK if there's a better fit somewhere else.

Reply:
Yep, this is a good spot.